dsl/src/main/kotlin/com/chutneytesting/kotlin/junit/engine/execution/ReportUtil.kt

```diff
@@ -28,10 +28,16 @@ object ReportUtil {
             .setType(step.type())
             .setTarget(step.target())
             .setStrategy(step.strategy().map { it.type }.orElse(null))
-            .setEnvironment(step.definition().environment)
+            .setEnvironment(getEnvironment(step))
```

Comment:
Same way that environment was removed from step definition, i would like to see the same here, i.e. environment removed from step report except for the root one.
Is it possible ?